services/rfq/relayer/chain/encoding.go

```
@@ -0,0 +1,83 @@
+	copy(result[offsetDeadline:offsetDeadline+sizeUint256], padUint256(tx.Deadline))
+	copy(result[offsetNonce:offsetNonce+sizeUint256], padUint256(tx.Nonce))
+
+	// Exclusivity address
+	copy(result[offsetExclusivityRelayer:offsetExclusivityRelayer+sizeAddress], tx.ExclusivityRelayer.Bytes())
+
+	// More uint256 values
+	copy(result[offsetExclusivityEndTime:offsetExclusivityEndTime+sizeUint256], padUint256(tx.ExclusivityEndTime))
+	copy(result[offsetZapNative:offsetZapNative+sizeUint256], padUint256(tx.ZapNative))
+
+	// Append ZapData
+	result = append(result, tx.ZapData...)
```

golangci-lint (1.62.2)

append to slice `result` with non-zero initialized length

(makezero)